packages/nameguard-react/src/components/Report/ReportFormattedName.tsx

```diff
@@ -1,26 +1,28 @@
 import React from "react";
+import { DisplayedName } from "../DisplayedName/DisplayedName";
+import { ENSName } from "@namehash/ens-utils";
+import { Normalization } from "@namehash/nameguard";
 
 type Props = {
+  name: ENSName;
   normalization: string;
-  name: string;
-  displayName: string;
+  beautifiedName?: string;
 };
 
 export function ReportFormattedDisplayName({
+  beautifiedName,
```

Comment:
Please see my comment above about how the props should change.

Please also note that `ENSName` does not have a `beautifiedName` field on purpose. It has a `displayName` field instead. That's because only normalized names can have a `beautifedName` but we always provide a `displayName` for all possible name values.

Please ask if any questions.